src/onegov/org/app.py

```diff
@@ -193,23 +193,25 @@ def ticket_permissions(self):
         return result
 
     @orm_cached(policy='on-table-change:pages')
-    def homepage_pages(self) -> dict[int, list[Topic]]:
-
+    def topics_cached(self) -> list[tuple[int, 'Topic']]:
```

Comment:
```suggestion
    def topics_cached(self) -> tuple[tuple[int, 'Topic'], ...]:
```
As a consequence from changing the return statement below